test/fixtures/templates/bad/noecho.yaml

```diff
@@ -0,0 +1,14 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  NoEchoParam:
+    Description: This shouldn't show
```

Comment:
```suggestion
```

Nitpick, but could be slightly confusing if this description is talking about the description itself instead of the value of the parameter, so maybe we can just drop it

Reply:
I'm certainly open to removing the statement, but am wondering what your thoughts are on instead changing the value to be more specific with something like "this value should not show".

Namely, I just want to ensure that any developer in the future can clearly see the reason behind the NoEcho warning without the need for significant context.

Reply:
also feels a bit misleading because the value is currently shown :)

Reply:
error message got bit by this same misunderstanding:
https://github.com/aws-cloudformation/cfn-python-lint/blob/9c4e023050771075d472bf3a246a534f54104fe1/src/cfnlint/rules/resources/NoEcho.py#L45